modules/restore-hosted-cluster-virt.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assembly:
+//
+// * hosted_control_planes/hcp_high_availability/hcp-backup-restore-virt.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="restore-hosted-cluster-virt_{context}"]
+= Restoring a hosted cluster on {VirtProductName}
+
+After you back up a hosted cluster on {VirtProductName}, you can restore the backup.
+
+[NOTE]
+====
+The restore process can be completed only on the same management cluster where you created the backup.
+====
+
+.Procedure
+
+. Ensure that no pods or persistent volume claims (PVCs) are running in the `HostedControlPlane` namespace.
+
+. Delete the following objects from the management cluster:
+
+** `HostedCluster`
+** `NodePool`
+** PVCs
```

Comment:
Are these pretty well understood by your user, or should the "how" be elaborated?

Reply:
If someone is completing this task, they'll likely know how to delete those objects.